login-mobile.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">

```

Comment:
Add 120-150 characters description in meta tag. @Prashantkankaria - plz provide a short description of the service (This description shows of on search results)
![screenshot from 2016-12-04 15-44-06](https://cloud.githubusercontent.com/assets/5563351/20865457/905fe40a-ba38-11e6-845d-5d0fd5fca616.png)